### PR TITLE
Candida18 Contributed a Word Cloud Generator 

### PR DESCRIPTION
**Pull request by Candida18**
# Description

## Word Cloud Generator
Word Cloud is a data visualization technique used for representing text data in which the size of each word indicates its frequency or importance. Significant textual data points can be highlighted using a word cloud. Word clouds are widely used for analyzing data from social network websites.

## Type of change

- [x] New Script (non-breaking change which adds a new script to the repository)
- [x] This change requires a documentation update

## Screenshot

<kbd></kbd>

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by OBITORASU** [comment]
Fix the merge conflict. Also add the linked issue in the PR description like this fixes #(issue number).

File: scripts/Word-Cloud-Generator/Bitcoin.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
+
+These characteristics make Bitcoin fundamentally different from a fiat currency, which is backed by the full faith and credit of its government. Fiat currency issuance is a highly centralized activity supervised by a nation’s central bank. While the bank regulates the amount of currency issued in accordance with its monetary policy objectives, there is theoretically no upper limit to the amount of such currency issuance. In addition, local currency deposits are generally insured against bank failures by a government body. Bitcoin, on the other hand, has no such support mechanisms. The value of a Bitcoin is wholly dependent on what investors are willing to pay for it at a point in time. As well, if a Bitcoin exchange folds up, clients with Bitcoin balances have no recourse to get them back.
```
Comment (OBITORASU) [inline scripts/Word-Cloud-Generator/Bitcoin.txt:3]: Have this file removed.

File: scripts/Word-Cloud-Generator/READme.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Word Cloud
+<p align="justify">Word Cloud is a data visualization technique used for representing text data in which the size of each word indicates its frequency or importance. Significant textual data points can be highlighted using a word cloud. Word clouds are widely used for analyzing data from social network websites.
```
Comment (OBITORASU) [inline scripts/Word-Cloud-Generator/READme.md:2]: Please add some documentation on how to run your script.

**Comment by OBITORASU** [comment]
Move your `.png` file to a dedicated images directory under your script directory. Also could you please convert your `.ipynb` file to a `.py` file? This [post](https://stackoverflow.com/questions/35545402/how-to-run-an-ipynb-jupyter-notebook-from-terminal) can help you with that, read the second answer.

**Review by OBITORASU — CHANGES_REQUESTED** [review]
Fix the merge conflict and check the other comments I left above.

**Comment by OBITORASU** [comment]
Could you please convert the `.ipynb` file in a `.py` file?

**Comment by OBITORASU** [comment]
Looks good just fix the merge conflict.

**Comment by rijusougata13** [comment]
hi, also add a screenshot in the PR

**Comment by Candida18** [comment]
> Fix the merge conflict and check the other comments I left above.

I have made the changes. Please close this comment

**Comment by rijusougata13** [comment]
Looks good ! Please resolve the conflict. I'll merge the PR

File: scripts/Word-Cloud-Generator/Word_Cloud.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
+"""
+Install the following modules
+
+pip install wordcloud
+pip install fileupload
+pip install ipywidgets
+jupyter nbextension install --py --user fileupload
+jupyter nbextension enable --py fileupload
+
+"""
```
Comment (rijusougata13) [inline scripts/Word-Cloud-Generator/Word_Cloud.py:10]: also put this instruction in readme so end user can find out whats to install. Change this and your PR is ready to merge

**Comment by rijusougata13** [comment]
great work !!

**Comment by Candida18** [comment]
> great work !!

Please add the necessary labels especially the `hacktoberfest-accepted` label. Since I've worked so hard.

**Comment by OBITORASU** [comment]
@Candida18 the label is not needed for your PR to count. Any PR which is merged counts towards your progress.

From Hacktoberfest 2021 FAQ, Rules section FAQ number 13
```
Your pull requests will count toward your participation if they are in a repository with the 'hacktoberfest 'topic and once they have been merged, approved by a maintainer or labeled as **hacktoberfest-accepted**. 


Additionally, any pull request with the **hacktoberfest-accepted** label, submitted to any public GitHub/GitLab repository, with or without the hacktoberfest topic, will be considered valid for Hacktoberfest.
```

**Comment by rijusougata13** [comment]
> > great work !!
> 
> Please add the necessary labels especially the `hacktoberfest-accepted` label. Since I've worked so hard.

Please wait couple of hours and you can find that progress in your hacktoberfest profile.

**Pull request merged**